src/realmGeolContinental.ttl

```diff
@@ -127,12 +132,18 @@ soreagcont:Craton rdf:type owl:Class ;
                               owl:onProperty sorelph:hasPlanetaryStructure ;
                               owl:allValuesFrom soreagcont:Shield
                             ] ;
-            rdfs:label "craton"@en .
+            dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+            dcterms:source <http://www.glossaryofgeology.org> ;
+            skos:definition "A craton is part of Earth's continental crust that has attained stability and has been little deformed for a prolonged period. Cratons include shield areas, where Precambrian rocks are exposed, and platform areas, where Precambrian rocks are overlain by a thin layer of Phanerozoic strata. (Neuendorf et al., 2011)"@en ;                            
+            skos:prefLabel "craton"@en .
```

Comment:
Restrictions on craton seem problematic:
```
  rdfs:subClassOf [      rdf:type owl:Restriction ;
      owl:allValuesFrom matrRockIgneous:Kimberlite ;
      owl:onProperty sorelch:hasSubstance ;     ]
```

would seem to imply that a craton may only consist of Kimberlite (or a subClass of Kimberlite?  That doesn't seem correct.
also there are three separate owl:allValuesFrom restrictions on  owl:onProperty sorelph:hasPlanetaryStructure, with different classes as the target.  I suspect this will lead to logical validation failure.  Without diving into the science question of are these actually the only planetary structures allowed on a craton, I think the restriction should be (in Turtle encoding):
```
rdfs:subClassOf [       rdf:type owl:Restriction ;
      owl:allValuesFrom [
          rdf:type owl:Class ;
          owl:unionOf (
              soreagb:IntracratonicBasin
              :ContinentalPlatform
              :Shield     ) ;  
        ] ;
      owl:onProperty sorelph:hasPlanetaryStructure ;
    ] ;

```

Reply:
We didn't contribute any restrictions like this, only definitions and sources, so we've not examined the logic of them. 

Please can we leave such discussion to a separate PR/Issue? There are enough things to deal with in this one already. Yes, I think the sort of change you suggest @smrgeoinfo is good but let's have that as another PR.

Reply:
I agree.